src/index.js

```diff
@@ -3,6 +3,8 @@ import displayTask from './module/displayTask.js';
 import adding from './module/add.js';
 import removetask from './module/remove.js';
 import edit from './module/edit.js';
+import check from './module/check.js';
+import removeAll from './module/removeAll.js';
 
 displayTask(JSON.parse(localStorage.getItem('todo')));
 
```

Comment:
- [ ] Dear @danielsafari143, Good job adding the UI elements, But Please remember for this milestone it's expected that adding and removing TODOs should be implemented and work as expected. Besides that, we need an event listener for the checkbox (`change`). The task object's values for `completed` is also expected to be updated upon user actions.
- [ ] Also, the updated array of items is expected to be stored in local storage, so the user gets the correct list values after the page reloads. 
**Please make sure all the functionalities are in place before the next review request.**

https://user-images.githubusercontent.com/69077061/236191519-00e7335e-8aac-41c7-be34-bfd42be3df45.mp4